### ISSUE TITLE
Allow disabling AutoThrottle bypassing

### ISSUE THREAD

**Issue by Gallaecio** [scrapy-plugins/scrapy-zyte-api#99]
The downloader middleware of scrapy-zyte-api was created to prevent AutoThrottle to affect requests driven through Zyte API, and instead let Zyte API itself control throttling on the server side, sending HTTP 429 responses when a spider is hitting a website too hard.

Relying on Zyte API to handle per-website throttling should most often be the best solution, since Zyte API can have a better picture of the traffic that a website can support and having central throttling control allows running multiple spiders against the same domain in parallel without increasing the overall concurrency to the upstream website.

However, some users might want to let AutoThrottle do its thing anyway. We could implement a setting to let them do just that.

**Comment by kmike** [comment]
Currently as a workaround users can use CONCURRENT_REQUESTS_... options; it should give similar results (see https://docs.scrapy.org/en/latest/topics/autothrottle.html#how-it-works), but not exactly the same.

**Comment by Gallaecio** [comment]
Do we still want this? I am not so sure anymore, however trivial it may be to implement.

**Comment by DharmeshPandav** [comment]
is there any update on this one?

I see the attached scrapy PR where it's allowing to disable autothrottle per request / spider by setting download_slot and throttle to false

Will there be a follow-up PR to fix zyte-API to make it respect auto_throttle? 

think this would be a useful feature as zyte-api is visiting the site at a rapid rate (close to 180 requests per minute) for a very long time without slowing down


I am told to use the `CONCURRENT_REQUESTS_PER_DOMAIN` and `CONCURRENT_REQUESTS_`but as @kmike  suggested its not the best way to handle it
